test/asm/align-large-ofs.err

```diff
@@ -1,3 +1,3 @@
 error: align-large-ofs.asm(2):
-    Alignment offset (2) must be smaller than alignment size (2)
+    Alignment offset must be between 0 and 1, not 2
```

Comment:
I think the old error message is clearer, as it states where the limit is coming from. What do you think?

Reply:
I'm not sure. The current error message is how two of the three copies were already. And if/when we allow negative offsets, it would need rephrasing anyway.

Reply:
Well, if those are allowed, then this error message would disappear entirely, so that wouldn't matter.

As for population counts, I'd blame copy-pasting what was nearby, and thus not considering over inherent explanation quality.

Reply:
Negative offsets would change the allowed range from [0, N) to (-N, N), with max N = 15. Some out-of-range values would still be possible and warned about.

Reply:
I will change it now though.